README.rst

```diff
@@ -2,6 +2,49 @@
 RestrictedPython
 ================
 
-RestrictedPython is a defined subset of the Python language which allows to provide a program input into a trusted environment.
+RestrictedPython is a tool that helps to define a subset of the Python language which allows to provide a program input into a trusted environment.
+RestrictedPython is not a sandbox system or a secured environment, but it helps to define a trusted environment and execute untrusted code inside of it.
 
 For full documentation please see  http://restrictedpython.readthedocs.io/ or the local ``docs/index``.
+
+Example
+=======
+
+To give a basic understanding what RestrictedPython does here two examples:
+
+An unproblematic code example
+-----------------------------
+
+Python allows you to execute a large set of commands.
+This would not harm any system.
+
+    >>> from RestrictedPython import compile_restricted
+    >>> from RestrictedPython import safe_builtins
+    >>>
+    >>> source_code = """
+    ... def example():
+    ...     return 'Hello World!'
+    ... """
+    >>>
+    >>> locals = {}
+    >>> byte_code = compile_restricted(source_code, '<inline>', 'exec')
+    >>> exec(byte_code, safe_builtins, locals)
```

Comment:
`locals` is not such a nice name here as it might be confused with the `locals` built-in. 
Even the code renderer of GitHub does this: it renders `locals` in blue but other variables in black.